src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75d;
+    private int threshold;
+    private int capacity;
+    private int size;
+    private Node<K, V>[] data;
+
+    public MyHashMap() {
+        capacity = DEFAULT_CAPACITY;
+        data = new Node[capacity];
+        threshold = (int) (capacity * LOAD_FACTOR);
```

Comment:
```suggestion
        threshold = capacity * LOAD_FACTOR;
```
this cast is redundant, but this is a minor recommendation
nothing critical